politeiawww/memorycache.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"sync"
+	"time"
+)
+
+// Item is a cached reference
+type Item struct {
```

Comment:
Since all of this is in package main, there's no point in exporting any of the structs or functions.

Reply:
done.